### PR TITLE
chore(arrays.js): clarify comment

### PR DESCRIPTION
**Pull request by henryw4k**
Made the comment clear by fixing the grammar.
Took @michaelficarra suggestion to make comment change even better.

### CODE REVIEW

**Comment by michaelficarra** [comment]
What happened to line 255? Also, please don't open multiple PRs next time. Just update the already open one.

**Comment by henryw4k** [comment]
opps accidentally deleted it. I added it back in the commit. Thanks for catching that. Will update PRs next time instead; just started trying to contribute to open source :P Thank you!

File: test/arrays.js [hunk 1]
```diff
@@ -251,7 +251,7 @@
     var stooges = _.zip(['moe', 30, 'stooge 1'], ['larry', 40, 'stooge 2'], ['curly', 50, 'stooge 3']);
     assert.deepEqual(stooges, [['moe', 'larry', 'curly'], [30, 40, 50], ['stooge 1', 'stooge 2', 'stooge 3']], 'zipped pairs');
 
-    // In the case of difference lengths of the tuples undefineds
+    //In the case of different lengths of the tuples, undefined values
```
Comment (michaelficarra) [inline test/arrays.js]: Comment must start with a space.

**Comment by henryw4k** [comment]
done

**Comment by michaelficarra** [comment]
Thanks.

**Pull request merged**